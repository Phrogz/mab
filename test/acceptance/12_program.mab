p( 'Test#12' )
p( 'program is:', program )