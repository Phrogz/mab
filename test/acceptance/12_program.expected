Test#12
program is:
p( "Test#12" )
p( "program is:", program asCode )